activitysim/abm/models/non_mandatory_tour_frequency.py

```diff
@@ -151,12 +151,9 @@ def non_mandatory_tour_frequency(persons, persons_merged, chunk_size, trace_hh_i
 
     model_settings = config.read_model_settings(model_settings_file_name)
 
-    # FIXME kind of tacky both that we know to add this here and del it below
-    # 'tot_tours' is used in model_spec expressions
     alternatives = simulate.read_model_alts(
         "non_mandatory_tour_frequency_alternatives.csv", set_index=None
     )
-    alternatives["tot_tours"] = alternatives.sum(axis=1)
```

Comment:
I would leave this line in place, but rather than simply assigning to "tot_tours": (1) check if that column name already exists, (2) write this sum if does not exist, and emit a FutureWarning while doing so to inform the user that ActivitySim may not auto-populate the "tot_tours" value in the future, and (3) emit a warning (not a show-stopping error) if there is any consistency problems between an exogenously defined "tot_tours" and the expected values.  This will allow existing model implementations to survive the code change without editing config files.